.github/dependabot.yml

```
@@ -11,3 +11,4 @@ updates:
   - gautam-yoti
```

And we can add you as a reviewer too
```suggestion
  - gautam-yoti
  - michalmilewiczyoti
```